setup.py

```diff
@@ -120,8 +120,7 @@
         'horovod': ['horovod[pytorch]>=0.19'],
         'funsor': [
             # This must be a released version when Pyro is released.
-            # 'funsor[torch] @ git+git://github.com/pyro-ppl/funsor.git@c685dde1746a8234b44a3aa0d6ab4e3a4d619c41',
-            'funsor[torch]==0.4.0',
+            'funsor[torch] @ git+git://github.com/pyro-ppl/funsor.git@master',
```

Comment:
Ah, sorry, to clarify, can you pin to the commit hash of the most recent commit on `master` (`5d737584ad055cdff8d56f38e0c3dd10d0e7e80e`), as in the commented-out line above? Pinning to a commit hash rather than a tag/branch keeps Funsor changes from causing test failures in other, unrelated Pyro PRs.